lib/utils.ts

```diff
@@ -0,0 +1,3 @@
+const delay = (ms: number): Promise<void> => new Promise((resolve) => setTimeout(resolve, ms));
+
+export default delay;
```

Comment:
Considering the name of this file is a generic `utils.ts`, it seems like this should actually be a named export rather than a default export to pave the way for more utility methods in the future, otherwise we'd need to update all imports of this later.

Reply:
Done!